tensorflow/core/util/cuda_kernel_helper.h

```diff
@@ -47,16 +129,22 @@ struct CudaLaunchConfig {
 // memory-limited.
 inline CudaLaunchConfig GetCudaLaunchConfig(int work_element_count,
                                             const GPUDevice& d) {
+  CudaLaunchConfig config;
+
+  // in case of invalid input, return the default value config, which has all -1
+  if (work_element_count <= 0) {
+    return config;
+  }
```

Comment:
The original code ends up creating a config with virtual_thread_count=0, I think. I'm guessing that the original author thought that returning such a config is fine in this case, since presumably the caller will subsequently call CUDA_1D_KERNEL_LOOP(i, n)   with n==0 which becomes a noop. @zheng-xq can you advice?